core/src/main/java/io/questdb/cairo/TableReaderMetadata.java

```diff
@@ -150,19 +153,22 @@ public void cloneTo(MemoryMA mem) {
     public void close() {
         Misc.free(metaMem);
```

Comment:
since `meteMem` is not longer `final` it can be assigned null when freed

Reply:
done in ed43dd5